components/common/layout.tsx

```diff
@@ -11,7 +11,7 @@ const Layout: FC = ({ children }) => {
         </Head>
         <Navbar />
         <main>{children}</main>
-    </>
-}
+    </>;
```

Comment:
Not a huge fan of these `;` at the end of jsx...but not sure how to get rid of them using current rules

- [ ] look into linting rule that works well for jsx